packages/services/api/src/modules/schema/providers/registry-checks.ts

```diff
@@ -483,19 +483,47 @@ export class RegistryChecks {
             return;
           }
 
+          let checkCoordinate = change.breakingChangeSchemaCoordinate;
+          if (isNullToRequiredArgumentChange(change)) {
+            // If this is a nullable argument changing to required, then we must count all calls -- even those not passing an argument.
+            // Otherwise, if those operations are not considered, they could still be getting sent. And since they are not passing the
+            // argument, they would be broken... This could likely be addressed by adjusting the coordinates collection logic.
```

Comment:
> This could likely be addressed by adjusting the coordinates collection logic.

I think we can not easily change the coordinate collection logic as we cannot control what version people are using. 🤔 

We can only add new features on top of the existing coordinate collection that people can use if they upgrade.

If we fundamentally change how it works, we would need to introduce versioning.

But I am not sure if this is relevant here.

Reply:
It is. This is exactly why i ultimately went thr route that i did